task_10/toma.lobanova/src/main/java/com/hillel/tasks/BinarySearch.java

```diff
@@ -10,7 +10,7 @@ private boolean exists(int[] array, int number, int startIndex, int endIndex) {
     if (endIndex < startIndex) {
       return false;
     }
-    int midIndex = startIndex + (endIndex - startIndex) / 2;
+    int midIndex = (startIndex + endIndex) >>> 1;
```

Comment:
Okay, thank you. I had just usual dividing by 2, but  i could't pass testing by travis because of it.  >>> was the advice from the report.